src/renderer/components/+catalog/catalog.tsx

```diff
@@ -269,7 +264,7 @@ export class Catalog extends React.Component<Props> {
           item.kind,
           item.source,
           item.labels.map((label) => <Badge key={label} label={label} title={label} />),
-          { title: item.phase, className: kebabCase(item.phase) }
+          { title: item.phase, className: cssNames({ [styles.connected]: item.phase == "connected" }) }
```

Comment:
Why did we change this? 🤔 I think the previous logic was more generic?

Reply:
Such cases now needs a bit different approach, because of use CSS Modules. We can't just add `connected` classname in here because all classnames from `catalog.module.css` will be converted to unique ones, so literal `connected` will never meet `styles.connected` class name.

Reply:
Basically previous logic can be untouched if using `*.scss` instead of `*.module.css` for styling.

Reply:
So for dynamic use cases we should use inline css?

Reply:
For dynamic use cases with a lot of different statuses we should specify which classnames could be used
```
{ title: item.phase, className: styles[kebabCase(item.phase) as "connected" | "disconnected"] }
```

Reply:
IMHO that's not dynamic if we need to define all possible phases. This is something we need to figure out at some point.

Reply:
I agree, should we do this in separate PR then?

Reply:
Why are `*.module.css` better than `*.scss`?

Reply:
`*.module.css` indicates this file is using CSS Modules and all it's classnames will be transformed to unique ones. If you're asking why using clear CSS instead of SCSS - currently it's the only way to work now (PostCSS isn't configured to use modules inside `scss` files). And it seems that we're okay using css for most of our tasks - we have native variables and nested css plugin.

Reply:
okay sounds good, I really like the nested-ness of `scss`. But if we can continue to do that then 👍 from me.

Reply:
@jakolehm Reverted to use dynamic classnames. It was made possible by some TS types fine-tuning. So, in files where you want to use such feature, you use `makeCss` utility:

```
const css = makeCss(styles);
```